### PR TITLE
{lang}[foss/2018b,intel/2018b] Python v3.6.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2604.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd7d8d4750762c989b1b71430a4199ad for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f0c63d65a30edb6e657301033bf9f072 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/57add74ebed2c95ca4e67994a4e7333c for a full test report.

**Comment by migueldiascosta** [comment]
can `numpy` be upgraded to `1.15.0`?

**Comment by akesandgren** [comment]
numpy 1.,15.0 would be preferable yes...

**Comment by akesandgren** [comment]
Ah, I though we had 1.15 in 3.7... I don't really care that much, 1.14.5 is at least the latest 1.14.x so it's good enough i think.

**Comment by migueldiascosta** [comment]
I do have something (`pymatgen`) that for some reason already requires 1.15, but I can work around that

**Comment by akesandgren** [comment]
Btw, being in sync with what extensions are used in Python 2.7.15 should not be a concern.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3123.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/752048b5707dc706ffb723e10b36fa34 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2614.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/025fc44979ff6d5776db1a8f5a2a096d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2056.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/33253f41d1c08affe08a38f2ff7e6738 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.5.1804, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/1fd14e9ee768eae09ffbb0f36794df93 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 20 out of 28 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/e17c3f6bcb95d41b3f849f0b74e42799 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/c1988c01b172b5327baa757ad7128752 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/c86a7cf802c9d95c4470c0422f78c535 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in this PR)
b-cn1522.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz, Python 2.7.12
See https://gist.github.com/bc3741d6b08f9588cc8f9c97a68738d1 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    # ('OpenSSL', '1.1.0h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+exts_download_dep_fail = True
+
+# workaround for "undefined symbol: __stack_chk_guard"
+# see also https://software.intel.com/en-us/forums/intel-c-compiler/topic/610514
+buildopts = 'LDFLAGS="$LDFLAGS -lssp"'
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb]: Is this really needed in the foss version or is it just convenient to keep the identical?

Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb]: The latter (and it doesn't hurt)

Comment (akesandgren) [reply easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb]: btw, where does ssp come from? Python itself?

**Comment by verdurin** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Python/Python-3.6.6-foss-2018b.eb]: No, `GCCcore`, see `lib64/libssp.so`

**Pull request merged**